### ISSUE TITLE
Is full inter-character spacing common for narrow columns in Tamil?

### ISSUE THREAD

**Issue by r12a** [w3c/i18n-activity#825]
**This is a tracker issue.** Only discuss things here if they are i18n WG internal meta-discussions about the issue. **Contribute to the actual discussion at the following link:**

§ https://github.com/w3c/iip/issues/58

**Comment by murasu** [comment]
Yes, it is, when the text needs to be justified in full. The preference is to have word-breaking so the space that needs to be covered will be minimised. Not all software provide proper word-breaking features in Tamil.
